scripts/systemd/serial-keel.template.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Serial Keel Server
+
+[Service]
+User=[user-name-here]
+WorkingDirectory=/home/[user-name-here]/serial-keel/
+ExecStartPre=/home/[user-name-here]/serial-keel/scripts/systemd/check_and_upgrade_serial_keel.sh [branch-here]
```

Comment:
Since we do `git rev-parse --show-toplevel`, we have the path to the repo.
Could we put that path here instead of `/home/[user-name-here]`?
Then the user can have the `serial-keel` folder anywhere.

Reply:
Changed the script so that we could have the serial keel folder anywhere